.travis/maybe-publish-coverage.sh

```diff
@@ -4,19 +4,19 @@ source $(npm root -g)/pr-bumper/.travis/is-bump-commit.sh
 
 if isBumpCommit
 then
-  echo "Skipping coverage publish for version bump commit"
+  echo "Skipping pr-bumper for version bump commit"
```

Comment:
I kept this line the same in the frost-core update: https://github.com/ciena-frost/ember-frost-core/pull/546/files#diff-872972d91442cb5b89f7fd8bebcf8a3eL7